projects/dev-schematics/src/shared/utils/file-utils.ts

```diff
@@ -0,0 +1,63 @@
+import { SchematicsException, Tree } from '@angular-devkit/schematics';
```

Comment:
After discussing with @tobi-or-not-tobi , we agreed to expose a very minimal `public_api` from _schematics_ project. 
Let's do that in this PR. 
At the same time, we can maybe refactor the method signatures a bit, in order to have one exposed "entry" function, to which we can pass all the required parameters.